lib/screens/app/app.dart

```diff
@@ -235,27 +235,29 @@ class _AppState extends State<App> with WidgetsBindingObserver {
   }
 
   Widget buildNavigation(bool showGuardianNotification) {
-    return BottomNavigationBar(
-      currentIndex: index,
-      onTap: (index) {
-        switch (index) {
-          case 0:
-            changePageNotifier.add("Wallet");
-            break;
-          case 1:
-            changePageNotifier.add("Explore");
-            break;
-          case 2:
-            changePageNotifier.add("Profile");
-            break;
-        }
-      },
-      backgroundColor: AppColors.primary,
-      items: navigationTabs
-          .map(
-            (tab) => buildIcon(tab.title, tab.icon, tab.iconSelected, tab.index == index, showGuardianNotification),
-          )
-          .toList(),
+    return Container(
+      decoration: BoxDecoration(border: Border(top: BorderSide(color: AppColors.white, width: 0.2))),
```

Comment:
should this not be 1 ? instead of 0.2? 
![image](https://user-images.githubusercontent.com/8041901/108129725-8e43dd80-707c-11eb-9bf7-216e3dab5a9d.png)

Reply:
no that would be the opacity = 0.2,  if I add 1 it does not look the same as the design